resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -177,17 +167,14 @@
 			"m_c avoids running into any cat, but r_c interrupts {PRONOUN/m_c/poss} peaceful silence.",
 			"m_c stares at r_c with an unreadable expression.",
 			"m_c tells a depressing anti-joke to r_c.",
-			"r_c asks how m_c's day is, but m_c's reply is kind of a bummer.",
 			"r_c learns the hard way that m_c isn't the best Clanmate to go to for advice.",
 			"m_c walks by r_c, mumbling something about a Clanmate with a sour expression.",
 			"r_c thinks it's funny how closely m_c protects {PRONOUN/m_c/poss} fresh-kill.",
-			"m_c tries to say something rude, but r_c just laughs at it.",
-			"r_c isn't phased by m_c's grumpiness today.",
-			"m_c snaps at r_c, but r_c knows it's just how {PRONOUN/m_c/subject} {VERB/m_c/are/is}."

```

Comment:
Removed

Reply:
Thought this one overlapped with other interactions in the same block, or were less interesting ^^